buildah/README.md

```diff
@@ -0,0 +1,63 @@
+# Buildah
+
+This Task builds source into a container image using Project Atomic's
+[Buildah](https://github.com/projectatomic/buildah) build tool. It uses
+Buildah's support for building from
+[`Dockerfile`](https://docs.docker.com/engine/reference/builder/)s, using its
+`buildah bud` command. This command executes the directives in the `Dockerfile`
+to assemble a container image, then pushes that image to a container registry.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/buildah/buildah.yaml
+```
+
+## Inputs
+
+### Parameters
+
+* **IMAGE**: The Docker image name to apply to the newly built image.
```

Comment:
Do you think it would make sense to switch this to an Output of type image instead of an input parameter? See https://github.com/tektoncd/catalog/pull/15 as well.

Reply:
Sure. I wasn't exactly sure how output images worked while they don't yet collect the pushed image digest (https://github.com/tektoncd/pipeline/pull/721), or whether I should use info about the Task's output image in the configuration of the steps. I guess that makes sense though, I'll give it a shot.